### PR TITLE
Add YOURLS_URL_CONVERT and YOURLS_NOSTATS env variable description

### PR DESCRIPTION
**Pull request by vitalyzhakov**

### CODE REVIEW

**Comment by tianon** [comment]
Is this feature already present in the published Docker image?

(cc @LeoColomb)

File: yourls/content.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The following environment variables are also honored for configuring your YOURLS
 -	`-e YOURLS_SITE=...` (yourls instance url)
 -	`-e YOURLS_USER=...` (yourls instance user name)
 -	`-e YOURLS_PASS=...` (yourls instance user password)
+-	`-e YOURLS_URL_CONVERT=..` (yourls URL shortening method: 36 or 62)
```
Comment (LeoColomb) [inline yourls/content.md]:
```suggestion
-	`-e YOURLS_URL_CONVERT=...` (yourls URL shortening method: `36` or `62`)
```

**Comment by LeoColomb** [comment]
@tianon Yes, but I'm not sure it's worth documenting since these variables do exactly what the official YOURLS documentation says (i.e. no specificity for the docker version).  
Let me know your opinion.

**Comment by tianon** [comment]
Agree, is there upstream documentation this could link to (or already is linking to)?

**Comment by LeoColomb** [comment]
Yes: https://yourls.org/#Config

**Comment by vitalyzhakov** [comment]
@tianon , @LeoColomb , hello!
I see, that [product config](https://yourls.org/#Config) has more options than configuration app via environment variables.
I think, it is a reason to update official docker hub docs.

**Comment by tianon** [comment]
I was thinking something more along the lines of this:

````diff
diff --git a/yourls/content.md b/yourls/content.md
index 36dee40f1..7bac58c41 100644
--- a/yourls/content.md
+++ b/yourls/content.md
@@ -16,7 +16,7 @@ $ docker run --name some-%%REPO%% --link some-mysql:mysql \
     -d %%IMAGE%%
 ```
 
-The following environment variables are also honored for configuring your YOURLS instance:
+YOURLS accepts [a number of environment variables for configuration](https://yourls.org/#Config). A few notable/important examples for using this Docker image include:
 
 -	`-e YOURLS_DB_HOST=...` (defaults to the IP and port of the linked `mysql` container)
 -	`-e YOURLS_DB_USER=...` (defaults to "root")
````

(Preferring to point to the upstream documentation instead of duplicating it.)

**Comment by LeoColomb** [comment]
@tianon That seems reasonable and useful. Should I make a PR or you prefer editing it directly?

**Comment by tianon** [comment]
Either way is fine with me -- PR away!

**Comment by LeoColomb** [comment]
Thanks a lot for this PR, @vitalyzhakov!  
However, as we have discussed here with @tianon, a short and reasonable documentation is preferable for the environment variables of the Docker version, referring to the official documentation for an extended version.

**Pull request closed without merging**